apps/i18n/common/en_us.json

```diff
@@ -792,9 +792,11 @@
   "levelsAttempted": "Levels attempted in",
   "levelStatus": "Level Status",
   "levelType": "Level Type",
+  "libraryCodeError": "We can’t publish your library because there is an error in the code. Go look for the red error indicator and fix the bugs.",
```

Comment:
Any suggestions? This error message was from Ryan during the pilot.

Oh also, I just noticed a smart quote there. Will fix that.

Reply:
Maybe something like, `red, square error indicator` - so there are two indicators of what to look for?

As far as 'error'/'bug' consistency - maybe just update `fix the bugs` to be `fix the errors`. That way be draw a clear line between 'Can't publish because of errors -> fix errors'.

Reply:
Maybe 
"We can’t publish your library because there is an bug in the code. Look for the square red error indicator and fix the bug."

Reply:
Haha looks like we responded at the same time. Thanks for the suggestion. I'll update it.